src/mscorlib/src/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -82,7 +82,7 @@ private unsafe int GetHashCodeOfStringCore(string source, CompareOptions options
 
             fixed (char* pSource = source)
```

Comment:
You can remove the bool flag for randomized string hashing while you are on it because of it is unused.